usecases/example1/HelloWorldSampleProject/K8sHelloWorldSample/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM wso2/micro-integrator:latest
+COPY CompositeApps/*.car /home/wso2carbon/wso2mi/repository/deployment/server/carbonapps/
+#COPY Libs/*.jar /home/wso2carbon/wso2mi/lib/
+#COPY Conf/* /home/wso2carbon/wso2mi/conf/
```

Comment:
@sajithaliyanage shall we please add a new line at EOF?

Applies to all cases in this PR.

Reply:
Done